OneBusAway_WatchKit_Extension/data/OBAStopGroupWK.h

```diff
@@ -0,0 +1,28 @@
+//
+//  OBAStopGroup.h
+//  org.onebusaway.iphone
+//
+//  Created by Shannon Young on 4/5/15.
+//  Copyright (c) 2015 OneBusAway. All rights reserved.
+//
+
+#import "OBAModelObjectWK.h"
+
+extern NSString *const OBARequestIdRecentAndBookmarkStopGroups;
+
+@class OBAStopWK;
+
+typedef NS_ENUM (NSInteger, OBAStopGroupType) {
+    OBAStopGroupTypeRecent,
+    OBAStopGroupTypeBookmark,
+    OBAStopGroupTypeNearby
+};
+
+@interface OBAStopGroupWK : OBAModelObjectWK
+
+@property (nonatomic) NSString *name;
```

Comment:
nitpick:  Wonder why not make the ownership explicitly defined.  There used to be a time when LLVM defaulted to assign.

In regards to NSString setters, it is good practices to make them copy, in case someone passes in an NSMutableString